client/src/components/Header.js

```diff
@@ -10,7 +10,8 @@ import menu from "assets/icons/menu.svg";
 import logo from "assets/logo.svg";
 import Logo from "./Logo";
 import { theme, mq } from "../constants/theme";
-
+import { Menu, Dropdown } from 'antd';
+import { DownOutlined } from '@ant-design/icons';
```

Comment:
Where are you using this `DownOutlined`? I think we can remove it since I don't see it anywhere.

Reply:
Good catch! it was actually used but then i double checked with the design and was not needed anymore! will remove now.